internal/mage/sdk/elixir.go

```diff
@@ -136,19 +136,28 @@ func (Elixir) Generate(ctx context.Context) error {
 	}
 	cliBinPath := "/.dagger-cli"
 
-	generated := elixirBase(c, elixirVersions[1]).
+	generated, err := elixirBase(c, elixirVersions[0]).
 		WithServiceBinding("dagger-engine", devEngine).
 		WithEnvVariable("_EXPERIMENTAL_DAGGER_RUNNER_HOST", endpoint).
 		WithMountedFile(cliBinPath, cliBinary).
 		WithEnvVariable("_EXPERIMENTAL_DAGGER_CLI_BIN", cliBinPath).
-		WithExec([]string{"mix", "dagger.gen"})
+		WithExec([]string{"mix", "run", "scripts/fetch_introspection.exs"}).
```

Comment:
It would be nice if mage tasks (or incoming ci module) could inject an introspection JSON file and let the codegen use that file.

Reply:
> That's done in modules, more specifically via the SDK runtime module. The server provides the introspection JSON and so, codegen needs to be able to read from that.

Yeah, I see it from the module runtime. So I thought it would be nice to have this included in generate task on non-support module SDK (rust, java, elixir, etc.). 

> An external way to get it is to:
> 
> ```
> dagger -m github.com/helderco/daggerverse/codegen@v0.1.2 call introspect as-json -o introspection.json
> ```

Ahh, so we can calling the function using `cliBinPath` to download introspection and then using it to generate code. Nice! Do we have a plan to include it in new CI module (#6843)?

Reply:
No plan to put it in the CI module, but you can do it for Elixir 🙂